### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_Keyboard:Typing in search form field and pressing enter key closes the migration assessment screen and no search results are displayed.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15855]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.2.2;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till Search form field.

### Actual:
Typing in search form field and pressing enter key closes the migration assessment screen and no search results are displayed.

### Expected:
When pressing enter key the results should be displayed or if there are no matching results a status message should be displayed as no results found.

### User Impact:
Keyboard only users will not be able to search anything in the screen.

**Comment by Harika1702** [comment]
<img width="863" alt="image" src="https://user-images.githubusercontent.com/100270074/218442119-697725d6-466e-403f-9353-962c4eacd093.png">
![15855_A11y_Issue fixed](https://user-images.githubusercontent.com/100270074/218442195-ba1f244a-7b17-499c-8907-408cafaea578.jpg)
Verified this issue in the below environment.
App name: Azure data Studio Insider.
Version: 1.42.0-insider (user setup)
Commit: 99a924dbcdce7bbaa0ab78b19f84e83035e54651
Date: 2023-02-08T06:58:10.882Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;
